tasks/us/census/spielman_singleton_segments.py

```diff
@@ -99,10 +94,7 @@ def convert_to_csv(self):
 
     def rename_files(self):
         rename = {
-            'ustractclustersnew-41530.bin' : 'US_tract_clusters_new.dbf',
-            'ustractclustersnew-41538.txt' : 'US_tract_clusters_new.prj',
-            'ustractclustersnew-41563.bin' : 'US_tract_clusters_new.shp',
-            'ustractclustersnew-41555.bin' : 'US_tract_clusters_new.shx'
+            'US_tract_clusters_new_20150720_427PM.shp': 'US_tract_clusters_new.shp',
```

Comment:
Maybe we should rename the file to a custom name to avoid a name change (but not content) that could lead to this task to fail. I know is useful to know if the file has changed but we should rely on a content hash and that is added to the roadmap in the `Data repository`

Reply:
This may work for most cases but it would fail now. The problem is that the downloaded zip contains  a malformed Shapefile: it contains a `prj`, a `shx` (which is correct), but two `shp` files, one partial and one total, and for both files the name is incorrect. In this case the only solution is to change the name of the `shp` file to match the rest of the Shapefile

Reply:
Ok understood